.code-samples.meilisearch.yaml

```
@@ -0,0 +1,967 @@
+# This code-samples file is used by the MeiliSearch documentation
+# Every example written here will be automatically fetched by
+# the documentation on build
+# You can read more on https://github.com/meilisearch/documentation/tree/master/.vuepress/code-samples
+---
+get_one_index_1: |-
+  client.getIndex(UID: "movies") { (result: Result<Index, Swift.Error>) in
+      switch result {
+      case .success(let index):
+          print(index)
+      case .failure(let error):
+          print(error)
+      }
+  }
+list_all_indexes_1: |-
+  client.getIndexes { (result: Result<[Index], Swift.Error>) in
+      switch result {
+      case .success(let indexes):
+          print(indexes)
+      case .failure(let error):
+          print(error)
+      }
+  }
+create_an_index_1: |-
+  client.createIndex(UID: "movies") { (result: Result<Index, Swift.Error>) in
+      switch result {
+      case .success(let index):
+          print(index)
+      case .failure(let error):
+          print(error)
+      }
+  }
+update_an_index_1: |-
+  client.updateIndex(UID: "movies") { (result: Result<(), Swift.Error>) in
+      switch result {
+      case .success:
+          print("Index updated")
```

```suggestion
  client.updateIndex(UID: "movies") { (result: Result<(Index), Swift.Error>) in
      switch result {
      case .success (let index):
          print(index)
```

As per the [documentation](https://docs.meilisearch.com/references/indexes.html#update-an-index)

---

I need to update the client.